craton/inventory/api/v1/cells.py

```
@@ -73,6 +73,23 @@ def post(self):
         cell = jsonutils.to_primitive(cell_obj)
         return cell, 200, None
 
+
+class CellById(base.Resource):
+
+    def get(self, id):
+        context = request.environ.get('context')
+        try:
+            cell_obj = dbapi.cells_get_by_id(context, id)
+        except exceptions.NotFound:
+            return self.error_response(404, 'Not Found')
+        except Exception as err:
+            LOG.error("Error during Cell get by id: %s" % err)
```

What circumstances could raise this catch-all exception handler? What testing could we do to raise to verify suitable responses?